celestia/chain.json

```diff
@@ -34,31 +32,26 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v2.1.2",
-    "compatible_versions": [
-      "v2.0.0",
-      "v2.1.0",
-      "v2.1.1",
-      "v2.1.2"
-    ],
+    "recommended_version": "v3.3.0",
+    "compatible_versions": ["v3.3.0", "v3.2.0", "v3.1.1", "v3.0.2", "v3.0.1"],
```

Comment:
We just released v3.3.1 as well.

Reply:
awesome, will add before marking ready for review, and thank you for the review

Reply:
https://github.com/cosmos/chain-registry/pull/5903/commits/942b010d2b867f2121fd9608d7217ce1e94fb53e